.github/workflows/build_devtool.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           uses: flutter-actions/setup-flutter@v2
           with:
             channel: stable
-            version: 3.16.9
+            version: 3.19.3
```

Comment:
can you make the version in `general.yml` exactly the same as the one in`build_devtool.yml` so that we are always certain this will pass if it passes CI?